src/components/enketo-iframe.vue

```diff
@@ -55,11 +60,11 @@ const invalidProps = computed(() => {
   return false;
 });
 
-const redirectUrl = computed(() => route.query.return_url);
+const redirectUrl = computed(() => route.query.return_url || route.query.returnUrl);
```

Comment:
Could we do a type check of these query parameters, checking that they're string? Vue Router allows query parameters to be array as well.